cla_backend/apps/call_centre/management/commands/create_and_assign_agilisys_organisation.py

```diff
@@ -0,0 +1,16 @@
+from django.core.management import BaseCommand
+from call_centre.models import Organisation, Operator
+
+
+class Command(BaseCommand):
+    help = "Create Agilisys organisation and assign current agilisys operators to that organisation"
+
+    def handle(self, *args, **options):
+        (organisation, created) = Organisation.objects.get_or_create(name="Agilisys")
+        operators = Operator.objects.filter(organisation__isnull=True, user__email__endswith="@agilisys.co.uk")
+        if not operators.count():
```

Comment:
Bit weird to me to have this in the negative. Think it'd read easier as `if` than `if not` (with the stanzas obviously reversed)